apps/codeforafrica/src/lib/data/blockify/ourWork.js

```diff
@@ -5,10 +5,14 @@ async function ourWork(block, api, context) {
   const data = await getProjects(api, query);
   const { docs: allProjects } = await api.getCollection("projects");
   const projectTags = allProjects.map(({ tag }) => tag).filter(Boolean);
-  const tags = [{ name: "All", slug: "all" }, ...new Set(projectTags)];
+  const tags = Array.from(new Set(projectTags.map((item) => item.id))).map(
+    (id) => {
+      return projectTags.find((item) => item.id === id);
+    },
+  );
```

Comment:
Since a tag's slug is unique, we can just use an object to reduce to unique tags without any search. We've done this before... check in the PRs that were merged last week I think.